org.azymohliad.Qwertone.json

```diff
@@ -0,0 +1,44 @@
+{
+    "app-id": "org.azymohliad.Qwertone",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "master",
+    "sdk": "org.gnome.Sdk",
+    "sdk-extensions" : [
+        "org.freedesktop.Sdk.Extension.rust-stable"
+    ],
+    "command": "qwertone",
+    "finish-args" : [
+        "--socket=x11",
```

Comment:
you need --share=ipc if you have this as well.

Reply:
thanks, done